app/process/roads_bikelanes/roadClassification/RoadClassification.lua

```diff
@@ -97,6 +107,10 @@ function RoadClassification(object)
     result_tags['road_oneway:bicycle'] = tags['oneway:bicycle']
   end
 
+  if minZoom11[result_tags.road] then
+    result_tags._minZoom = 11
```

Comment:
```suggestion
    result_tags._minzoom = 11
```

Damit es konsistent ist mit maplibre.